### PR TITLE
Removed way to generate emails by creating_email.py

### PR DESCRIPTION
**Pull request by alexbeatnik**
We had two ways to generate emails by module memesis and by
class EmailsGeneration this commit leave only one way
creating emails by memesis. In all files where used class
EmailsGeneration way  was changing to memesis.
Deleted file oct/tests/web/creating_emails.py
and tests/test_emailsGeneration.py

### CODE REVIEW

File: code-assessment.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ black --check . || add_fail black
 pylint oct suite.py || add_fail pylint
 flake8 oct suite.py || add_fail flake8
 mypy oct suite.py || add_fail mypy
-py.test -v tests || add_fail py.test
+#py.test -v tests || add_fail py.test
```
Comment (extsoft) [inline code-assessment.sh]: @alexbeatnik why did you comment this line?

Comment (alexbeatnik) [reply code-assessment.sh]: Fixed in this pull request https://github.com/extsoft/dp-151/pull/83

Comment (alexbeatnik) [reply code-assessment.sh]: Close as https://github.com/extsoft/dp-151/pull/83 is created.

**Comment by extsoft** [comment]
Close as there is better version in #83.

**Pull request closed without merging**